css/desktop.css

```diff
@@ -0,0 +1,309 @@
+body {
+  display: grid;
+}
+
+main {
+  width: 100vw;
+}
+
+.menu-icon {
+  display: none;
+}
+
+.main-header {
+  height: 72px;
+  padding: 16px 142px;
+}
+
+.dev-name {
+  padding: 10px 12px;
+  font-size: 20px;
+  line-height: 20px;
+  width: 116px;
+  font-family: "Inter", sans-serif;
+  text-align: center;
+  vertical-align: middle;
+  letter-spacing: 0.03em;
+}
+
+nav ul {
+  display: flex;
+  column-gap: 20px;
+}
+
+nav li {
+  padding: 10px 12px;
+  text-align: center;
+  vertical-align: center;
+}
+
+nav a {
+  text-decoration: none;
+  color: #fff;
+  font-family: "Inter", sans-serif;
+  font-style: bold;
+  font-size: 20px;
+  line-height: 20px;
+  line-height: 110%;
+  text-align: center;
+  vertical-align: middle;
+}
+
+.headline {
+  height: 870px;
+  background-image: url("../image/desktop.png");
+  background-repeat: no-repeat;
+  background-size: fill;
+  background-position: right 1px top 1px;
+  display: grid;
+  grid-template-columns: repeat(12, 1fr);
+  column-gap: 87px;
+  padding: 0 36px;
+}
+
+.headline-top-section {
+  display: flex;
+  flex-direction: column;
+  width: 560px;
+  padding: 192px 0;
+}
+
+.social-media {
+  order: -1;
+  align-self: flex-start;
+  display: flex;
+  flex-direction: column;
+  padding: 360px 0;
+  justify-content: center;
+}
+
+.title {
+  font-weight: 400;
+  font-size: 72px;
+  line-height: 81.8px;
+}
+
+.supporting {
+  font-size: 32px;
+  font-style: medium;
+  letter-spacing: 0.03em;
+  line-height: 44px;
+}
+
+.description {
+  font-size: 20px;
+  line-height: 32px;
+  font-style: regular;
+  letter-spacing: 0.03em;
+  font-weight: 400;
+}
+
+.work {
+  margin: 0 118px;
+  padding: 0 24px;
+}
+
+.work-header {
+  display: flex;
+  justify-self: start;
+  justify-content: center;
+  align-items: baseline;
+  column-gap: 24px;
+  margin-top: 135px;
+  margin-bottom: 80px;
+}
+
+.work-header .title {
+  font-size: 40px;
+  margin: 10px 10px;
+}
+
+.work-header hr {
+  width: 794px;
+  margin: 8px 0;
+  height: 1px;
+  padding-right: 100px;
+}
+
+.feature-section {
+  display: flex;
+  flex-wrap: wrap;
+  column-gap: 24px;
+}
+
+.feature-image {
+  width: 50%;
+}
+
+.feature-image img {
+  width: 100%;
+}
+
+.story-section {
+  grid-column: 8/13;
+  width: 448px;
+  display: flex;
+  flex-direction: column;
+  row-gap: 24px;
+}
+
+.story-title {
+  font-size: 45px;
+  line-height: 52px;
+}
+
+.story-description {
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 28px;
+  color: #091e42;
+}
+
+.card-section {
+  display: flex;
+  flex-wrap: wrap;
+  column-gap: 24px;
+}
+
+.card {
+  width: 369px;
+  height: 435px;
+  padding: 8px 16px;
+}
```

Comment:
- [ ] Your project cards look fine 👍. however, I noticed that there is something that changed due to the changes you made after the first review. Before the first review, your card section was in the 3-column grid but after you have made some changes as requested by the previous reviewer, your card section is in the 2-column grid which when you expanded the screen can even be a 4-column grid.
**Before the first review**
![Screenshot (91)](https://user-images.githubusercontent.com/90222110/177930021-9f0e625e-c43e-47dc-a303-2b19aea3e2ac.png)
**After doing some changes as suggested by the previous reviewer**
![Screenshot (92)](https://user-images.githubusercontent.com/90222110/177930284-41a2d1e2-1f8c-482e-b2a4-3339771c67de.png)
Please try to go through what you have changed and find out what is the cause so that you set it back to how it was. 
 💡  **The card section should be a 3-column grid on the desktop screens**

Reply:
@BertrandConxy Thank for your constructive feedback. But the card layout is in 3-column grid in my browser, please find out the image below
![2022-07-08_09-59-39](https://user-images.githubusercontent.com/30173722/177936421-537a9cc5-dfd2-4e08-93eb-66c1d48dd8ba.png)
.

Reply:
@BertrandConxy I use flex box instead of grid to become the column flexible. can I have to change it to grid? 
`.feature-section {
  display: flex;
  flex-wrap: wrap;
  column-gap: 24px;
}`

Reply:
Sure. Go ahead  :+1: